.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Type of Issue:
+Some example types are:
+* bug in science or answers
+* bug in software
+* science enhancement (changes answers and requires science development)
+* software enhancement
+* FATES change
+* Discussion
+
+
+## Summary of Issue:
+
+## Details of Issue:
+(Be sure to give enough details that the issue can understood sufficiently to replicate
+ or to make the required changes)
+
+# For bugs be sure to fill out the following (otherwise remove):
+-----------------------------------------------------------------
+# Expected behavior and actual behavior:
```

Comment:
With manage_externals, I've found this section on expected/actual behavior to be more bothersome than helpful: In some cases it doesn't apply, and in others it duplicates what was already given in the details of issue section.